Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/agenda/AgendaDayPagerAdapter.kt

```diff
@@ -18,9 +19,15 @@ class AgendaDayPagerAdapter internal constructor(
     }
 
     override fun getItem(position: Int): Fragment {
+        val dayString = if (position == 0) {
+            BuildConfig.EVENT_DAY_ONE_STRING
```

Comment:
its probably better to work with date/time objects and just use the BuildConfig to set value integers, do we use Joda time or Java.time in the app already?

Reply:
Not that I'm aware of. Definitely not Joda. Most places we deal with times are in strings lol

Do you mean to just have the build config set year/month/day, and then create calendar instances like we do in `AgendaFragment`?

```
val dayTwo = Calendar.getInstance()
            dayTwo.set(
                BuildConfig.EVENT_YEAR,
                BuildConfig.EVENT_MONTH - 1, // Calendar is 0 indexed
                BuildConfig.EVENT_DAY_TWO
            )
```

Reply:
The conference days can actually be pulled out of the DB...that info was not available the first year which is where that hardcoded junk came from :-(

Reply:
Specifically \conferenceData\Sections\<id>\startTime and endTime

Reply:
Aren't the conference days necessary to know which items to put in which tab? Unless there's a better way to do this. Right now it looks like we query for all schedule items, and then filter based on day client side.

In any case, we'll need some concept of the conference days app side. Which is why I was mostly interested in creating a single source of truth for them, which will be a lot smoother than trying to hunt them down and change them all.

Reply:
yep...see screencap of those DB nodes:
![screen shot 2019-03-06 at 11 39 21 am](https://user-images.githubusercontent.com/586670/53897644-95df5880-4004-11e9-9af2-dbd32c8c215d.png)


Yours is a good first step, but eventually, we can grab them out of the DB is all I was saying...

Reply:
Ah I see. Should we make a ticket for that? Or do you think that's a concern for after next month's event?

Reply:
It's not harmful to use the hardcoded values...just wanted to mention why it was there historically...
Reading it from the DB would be a nice to have...and below something more useful like session ratings...

Reply:
I appreciate the mention, since I don't have a lot of historical context. :) 

My question then is whether we should make an issue to follow up with later? If not are we good with this PR?

Reply:
This PR is good.  We can make an issue to follow up later if you want, but it's not a huge priority IMHO.